TelemetryProcessors/PreAggregatedMetricsTelemetryProcessor.ts

```diff
@@ -0,0 +1,53 @@
+import Contracts = require("../Declarations/Contracts");
+import AutoCollecPreAggregatedMetrics = require("../AutoCollection/PreAggregatedMetrics");
+import * as TelemetryType from "../Declarations/Contracts";
+import {
+    MetricDependencyDimensions,
+    MetricExceptionDimensions,
+    MetricRequestDimensions,
+    MetricTraceDimensions
+} from "../Declarations/Metrics/AggregatedMetricDimensions";
+import Context = require("../Library/Context");
+
+export function preAggregatedMetricsTelemetryProcessor(envelope: Contracts.EnvelopeTelemetry, context: Context): boolean {
+    // Increment rate counters
+    switch (envelope.data.baseType) {
+        case TelemetryType.TelemetryTypeString.Exception:
+            let exceptionDimensions: MetricExceptionDimensions = {
+                cloudRoleInstance: envelope.tags[context.keys.cloudRoleInstance],
+                cloudRoleName: envelope.tags[context.keys.cloudRole],
+            };
```

Comment:
`exceptions/count` also specifies a `client/type` field in the required dimensions. Is that needed here? What is dot net sdk doing?

Reply:
.Net doesn't send this dimension in pre aggregated metrics and we don't support that property in node.js SDK at all, is this something available in other SDKs?

Reply:
If .Net doesn't send this it's fine to leave it out.